### ISSUE TITLE
nemo-compare error in .xsession-errors

### ISSUE THREAD

**Issue by ChooChooAl** [linuxmint/nemo-extensions#519]
Got this error in log.   I include the java script line, not sure it's part of traceback or not.

JavaScript error: resource://services-settings/Utils.sys.mjs, line 245: TypeError: Headers.set: cf_use_ob=443; path=/; expires=mon, 25-mar-24 14:55:26 gmt is an invalid header name.
Traceback (most recent call last):
  File "/usr/share/nemo-python/extensions/nemo-compare.py", line 168, in get_name_and_desc
    description = _("Allows file comparison from the context menu")
NameError: name '_' is not defined

It's only once in the past 1day 17hours.    I don't know how to replicate it, just thought I'd leave it more for an FYI.

**Comment by leigh123linux** [comment]
Duplicate of #513
